client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -17,6 +17,12 @@
                             </li>
                         </ul>
                     </oui-tile-description>
+                    <oui-action-menu data-compact data-align="end">
+                        <oui-action-menu-item
+                            data-text="{{ ::'status_action_renew_management' | translate }}"
```

Comment:
`text` attribut is now deprecated. I roughly encourage you to use the transclude slot like this:

```html
<oui-action-menu
    data-compact
    data-align="end">
    <oui-action-menu-item data-href="{{ ::$ctrl.actions.manageAlldom.href }}">
        <span data-ng-bind="{{ ::'status_action_renew_management' | translate }}"></span>
    </oui-action-menu-item>
</oui-action-menu>
```

Reply:
Fixed by using span +  data-translate